Sources/Jobs/JobQueue.swift

```diff
@@ -41,11 +42,17 @@ public struct JobQueue<Queue: JobQueueDriver>: Service {
     ///   - id: Job identifier
     ///   - parameters: parameters for the job
     /// - Returns: Identifier of queued job
-    @discardableResult public func push<Parameters: Codable & Sendable>(id: JobIdentifier<Parameters>, parameters: Parameters) async throws -> Queue.JobID {
+    @discardableResult public func push<Parameters: Codable & Sendable>(
+        id: JobIdentifier<Parameters>, parameters: Parameters
+    ) async throws -> Queue.JobID {
         let jobRequest = JobRequest(id: id, parameters: parameters)
         let buffer = try JSONEncoder().encodeAsByteBuffer(jobRequest, allocator: self.allocator)
         let id = try await self.queue.push(buffer)
-        self.handler.logger.debug("Pushed Job", metadata: ["_job_id": .stringConvertible(id), "_job_type": .string(jobRequest.id.name)])
+        Counter(label: "queued_jobs_counter", dimensions: [("queue_name", jobRequest.id.name)]).increment()
```

Comment:
queue_name is a little misleading, shouldn't this be job_name, or JobName to coincide with the log